### DISCUSSION TITLE
tensorboard error

### DISCUSSION THREAD

**Discussion by stevencreed** [datawhalechina/thorough-pytorch#21] [Q&A] [answered]
- [ tensorboard无法打开网页，原因是安装了多个版本的tensorboard；
解决方法是 运行检测脚本 https://raw.githubusercontent.com/tensorflow/tensorboard/master/tensorboard/tools/diagnose_tensorboard.py ，按提示删除tensorflow tensorboard tb-lightly等，重新安装即可。]

**Comment by ZhikangNiu** [comment, marked as answer]
你的回答对于我们的教程很有帮助！

**Reply by ZhikangNiu** [reply]
运行检测脚本 https://raw.githubusercontent.com/tensorflow/tensorboard/master/tensorboard/tools/diagnose_tensorboard.py ，按提示删除tensorflow tensorboard tb-lightly等，重新安装即可。]
